ykrt/src/compile/jitc_yk/codegen/reg_alloc/mod.rs

```diff
@@ -14,15 +16,33 @@ pub(crate) enum VarLocation {
         /// Size in bytes of the allocation.
         size: usize,
     },
+    Direct {
```

Comment:
Can we document `Direct` and `Indirect`? At face value they seem identical?

Reply:
[9e8f08d](https://github.com/ykjit/yk/pull/1318/commits/9e8f08db1489fa33578063838ef651345903fcf1)